news/_posts/2020-03-bug-bounty-program.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "Let the hunt begin"
+subtitle: "It's time to make PrestaShop better that it already is!"
+date:   2020-03-11 08:00:00
+authors: [ PierreRambaud, PrestaShop ]
+icon: icon-chat
+image: /assets/images/theme/meta-logo-build.png
+tags: [bug, bounty, program, maintenance]
+---
+
+Protecting the privacy of PrestaShop's users is one of our top priority. We build the software with this goal in mind.
+That's why we decided to welcome help from the outisde through a bounty program to put our security to the test!
+
+# Why this program
+
+Bug bounties are used by many leading companies to improve the security of their products.
+These programs provide an incentive for researchers to responsibly disclose software bugs and allow security teams to
+leverage the external community to help keep users safe.
+
+Cyber criminals from around the world are continuously finding new ways of breaking websites or stealing
+personal information and more if they are related to ecommerce.
+
+By creating this program, and even if it may appear as digging our own grave, it is more about prevention.
+It invites White Hat Hacker to play with us and report issues before a "bad guy" messes up.
+
+# What's in scope
+
+First of all, keep in mind this bounty programm does not concern regular bugs, but only security flaws.
+If you have problems and wish to report a bug, feel free to open an issue on our GitHub repository [PrestaShop/PrestaShop](https://github.com/PrestaShop/PrestaShop/issues/new/choose)
+
+Then, this bounty program covers code from our GitHub repository [PrestaShop/PrestaShop](https://github.com/PrestaShop/PrestaShop).
+
+We are particularity interested in Remote Code Execution, SQL Injections, Authentication bypass, and obviously XSS with demonstrable business impact.
+
+Also, we consider non-qualitfying vulnerabilities for any [CVSS3](https://www.first.org/cvss/calculator/3.0) score lower than 4.0, unless it can be combined with other vulnerabilities to achieve a higher score.
+
+Find more information about qualifying and non-qualifying vulnerabilities and of course everything about Vulnerability disclosure on our [YesWeHack Public Program](https://yeswehack.com/programs/prestashop)!
```

Comment:
```suggestion
Find more information about qualifying and non-qualifying vulnerabilities and of course everything about Vulnerability disclosure on PrestaShop's [YesWeHack Public Program](https://yeswehack.com/programs/prestashop)!
```

Reply:
```suggestion
Find more information about qualifying and non-qualifying vulnerabilities and of course everything about vulnerability disclosure on [PrestaShop's YesWeHack Public Program](https://yeswehack.com/programs/prestashop)!
```

Reply:
Maybe finish the article with something like "Happy hunting!"